ee_tests/package.json

```diff
@@ -46,10 +47,9 @@
     "protractor-jasmine2-screenshot-reporter": "0.3.5",
     "rimraf": "^2.6.2",
     "ts-helpers": "1.1.2",
-    "ts-node": "3.0.2",
-    "tslint": "5.0.0",
+    "tslint": "5.8.0",
     "tslint-loader": "^3.5.0",
-    "typescript": "^2.3.0"
+    "typescript": "^2.6.0"
```

Comment:
we are currently using 2.5.3 every where else

Reply:
@joshuawilson thanks for the alert. I will submit a fix soon.

Reply:
@joshuawilson and should the version of node.js be 8.3.x ?

Reply:
yes, we are on Node v8.3.0 and npm v5.3.0

Reply:
@joshuawilson PR https://github.com/fabric8io/fabric8-test/pull/328 should fix this. Could you please review?